addons/api/addon/serializers/credential.js

```diff
@@ -5,6 +5,10 @@ export default class CredentialSerializer extends ApplicationSerializer {
   serialize() {
     const serialized = super.serialize(...arguments);
     delete serialized['attributes']['password_hmac'];
+    if (serialized.type === 'username_password') {
```

Comment:
Should this possibly be 
```suggestion
    if (serialized.type !== 'ssh_private_key') {
```
that way the check is flipped and if you add more types you don't have to keep adding to the if statement?

Reply:
This will likely change to a switch statement because we will have to do a similar delete on fields. Which, I believe we have an existing pattern of checking the type, then calling another serializer just for that type. I believe host catalogs do this. I am going to work on the serializer for this next and that should fix us this logic. This was mostly done to satisfy tests that failed due to the new attributes.